spec/linter-eslint-spec.js

```diff
@@ -367,24 +380,14 @@ describe('The eslint provider for Linter', () => {
     let editor
     let didError
     let gotLintingErrors
-    let tempFixtureDir
-    let tempFixturePath
+    const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
```

Comment:
I passed it in so that I could rimraf it after the test. Guess I could indeed rename it.